docs/agent-reference/how-to-run-agent.md

```diff
@@ -8,12 +8,24 @@ AducIotAgent '<IoT device connection string>' [options...]
 
 ### Additional Arguments
 
-#### --EnableIotHubTracing
+#### --version
+
+With this option enabled, the reference agent will output the version.
+
+#### --enable-iothub-tracing
 
 With this option enabled, the reference agent will output verbose tracing from
 the Azure IoT C SDK. This option is useful for troubleshooting connection
 issues.
 
+#### --health-check
+
+With this option enabled, the reference agent will turn on Health Check Only mode.
```

Comment:
super nit: as above so below

Reply:
[Malmsteen fan?](https://www.youtube.com/watch?v=LxmZawYJo74)

---
In reply to: [612710813](https://github.com/Azure/iot-hub-device-update/pull/39#discussion_r612710813) [](ancestors = 612710813)

Reply:
@JeffMill Haha no. My grandmother used it all the time when I was a kid. It's from the faerie mythology.

---
In reply to: [612718233](https://github.com/Azure/iot-hub-device-update/pull/39#discussion_r612718233) [](ancestors = 612718233,612710813)